src/main/kotlin/game/blackjack/domain/Dealer.kt

```diff
@@ -1,26 +1,24 @@
 package game.blackjack.domain
 
-class Dealer : Player("딜러") {
+class Dealer : Player("딜러", 0) {
     private val deck = Deck()
 
     fun drawCard(): Card = deck.draw()
-
-    fun recordWin() {
-        winningRecord.recordWin()
-    }
-
-    fun recordLose() {
-        winningRecord.recordLose()
+    fun drawCard(count: Int): List<Card> {
+        val drawn = mutableListOf<Card>()
+        repeat(count) { drawn.add(drawCard()) }
+        return drawn
```

Comment:
```suggestion
        return (1..count).map { drawCard() }
```
이런 형태는 어떠신가요. 사소한 부분이지만 함수형 언어의 장점을 활용하는 부분인 것 같아요.